commonjs-extension-resolution-loader/README.md

```diff
@@ -0,0 +1,16 @@
+Node's ESM specifier resolution does not support all default behavior of
+the CommonJS loader. One of the behavior differences is automatic resolution
+of file extensions and the ability to import directories that have an index
+file.
+
+Use this loader to enable automatic extension resolution and importing from
+directories that include an index file.
+
+```console
```

Comment:
I would add this after line 8 (can’t use the suggestion feature because of the code block inside):

<hr>

This loader enables code like this:

```js
import file from './file'; // Where ./file is ./file.js
import index from './folder'; // Where ./folder is ./folder/index.js
```

This loader also applies these automatic resolution rules to the program entry point passed to `node` on the command line: